lib/api/transactions.js

```diff
@@ -227,13 +227,16 @@ module.exports = function (app) {
 	};
 
 	this.getUnconfirmedTransactions = function (transactions, error, success) {
+		let unconfirmedUrl = '/node/transactions/unconfirmed';
+		if (app.get('node.majorVersion') === '2') unconfirmedUrl = '/node/transactions/ready';
```

Comment:
This endpoint contains only transactions that are ready to be included in the next block (max 25). There is another endpoint `/api/node/transactions/received` which returns all the pending transactions - would be more reasonable to use this one.